### DISCUSSION TITLE
How to disable edge routing points

### DISCUSSION THREAD

**Discussion by sathya-1994** [eclipse-glsp/glsp#747] [Q&A] [answered]
I am using vscode version of GLSP

When we select any edge, we will get edge routing points. But I want disable these edge routing points.

How to implement my requirement?

Screenshot:
![image](https://user-images.githubusercontent.com/59145404/191675830-cf9ce577-db5a-4fd6-9e23-3fca56d841e6.png)

**Comment by planger** [comment, marked as answer]
I think you can set `routable` to `false` in the [`EdgeEditHints`](https://github.com/eclipse-glsp/glsp/blob/a16967e126a6863784bd9d5342192585b53e1a40/PROTOCOL.md#212-element-type-hints) in your diagram configuration on the server.

Also, I think, you can disable the `editFeature` for the edge type in your GLSP client module, which would turn off those routing points edges of that type on the client.

```typescript
configureModelElement(context, 'your-edge-type-id', SEdge, YourView, { disable: [editFeature] });
```

**Reply by sathya-1994** [reply]
@planger ,
Server side EdgeEditHints did not help. **But client side disable: [editFeature] worked.**
